.vscode/settings.json

```
@@ -5,5 +5,6 @@
 	"editor.defaultFormatter": "esbenp.prettier-vscode",
 	"editor.formatOnSave": true,
 	"eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
+	"eslint.probe": ["javascript", "javascriptreact", "json", "jsonc", "markdown", "typescript", "typescriptreact"],
```

Sorry, one more tool 😄 #285 added support for the `"yaml"` file in linting to this repo:

```suggestion
	"eslint.probe": ["javascript", "javascriptreact", "json", "jsonc", "markdown", "typescript", "typescriptreact", "yaml"],
```

---

The prettier changes happened sort of because the first time I tried sending the PR I didn't clone locally, but through github.dev. Hope this fixes things! Have a happy weekend or great start to your week!

---

Thanks, likewise on the weekend & week! ❤️